File: integration-tests/src/test/java/stubidp/test/integration/support/TestUserCredentials.java
package stubidp.test.integration.support;

import org.mindrot.jbcrypt.BCrypt;
import stubidp.stubidp.configuration.UserCredentials;

public class TestUserCredentials extends UserCredentials {

    public TestUserCredentials(String user, String password) {
        this.user = user;
        this.password = BCrypt.hashpw(password, BCrypt.gensalt());
    }
}

File: integration-tests/src/test/kotlin/stubidp/kotlin/test/integration/support/TestUserCredentials.kt
package stubidp.kotlin.test.integration.support

import org.mindrot.jbcrypt.BCrypt
import stubidp.stubidp.configuration.UserCredentials

class TestUserCredentials(user: String?, password: String?) : UserCredentials() {
    init {
        this.user = user
        this.password = BCrypt.hashpw(password, BCrypt.gensalt())
    }
}